source/stylesheets/site/_utilities.scss

```diff
@@ -0,0 +1,5 @@
+.u-hide-small-screen {
```

Comment:
Are we namespacing all classes or just utility classes? Should we begin to document these architectural decisions somewhere, like in the readme?